internal/provisioner/plain/manifests/07_webhook.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: rukpak-webhook
+  namespace: rukpak-system
+spec:
+  ports:
+    - port: 443
+      protocol: TCP
+      targetPort: 9443
+  selector:
+    app: plain-provisioner
+
+---
+
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: rukpak-webhook
+  annotations:
+    cert-manager.io/inject-ca-from: rukpak-system/rukpak-webhook-certificate
+webhooks:
+- name: core.rukpak.io
+  admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: rukpak-webhook
+      namespace: rukpak-system
+      path: /validate-core-rukpak-io-v1alpha1-bundle
+      port: 443
+  failurePolicy: Fail
+  #name: vbundle.kb.io
+  rules:
+  - apiGroups:
+    - core.rukpak.io
+    apiVersions:
+    - v1alpha1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - bundles
+    #- "*/*"
+  sideEffects: None
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: rukpak-webhook-certificate
+  namespace: rukpak-system
+spec:
+  secretName: rukpak-webhook-certificate
+  dnsNames:
+  - core.rukpak.io
```

Comment:
Do we need the cert to have the `core.rukpak.io` DNS name?

Reply:
I'm not sure.  The runtime-controller might the group name as the domain name or the name of the webhook.  I'll figure it out.

Reply:
I took it out.  It should be `[service name].[namespace name].svc`